packages/mdc-fab/mdc-fab.scss

```diff
@@ -22,7 +22,7 @@
   @include mdc-fab-base_;
   @include mdc-fab-container-color(secondary);
   @include mdc-fab-ink-color(text-primary-on-secondary);
-  @include mdc-ripple-color(text-primary-on-secondary, $mdc-ripple-pressed-dark-ink-opacity);
+  @include mdc-states(text-primary-on-secondary);
 
   &:not(.mdc-ripple-upgraded) {
     -webkit-tap-highlight-color: rgba(0, 0, 0, .18);
```

Comment:
Is there are reason this remains a literal value and does not use the `mdc-theme-prop()` mixin?